pkg/genericapiserver/server/options/server_run_options.go

```diff
@@ -66,7 +66,7 @@ type ServerRunOptions struct {
 func NewServerRunOptions() *ServerRunOptions {
 	return &ServerRunOptions{
 		AdmissionControl:            "AlwaysAdmit",
-		DefaultStorageMediaType:     "application/json",
```

Comment:
This may not be appropriate, not all generic servers are required to support protobuf.